support/ebpf/sched_monitor.ebpf.c

```diff
@@ -6,29 +6,26 @@
 
 #include "types.h"
 
-// tracepoint__sched_process_exit is a tracepoint attached to the scheduler that stops processes.
-// Every time a processes stops this hook is triggered.
-SEC("tracepoint/sched/sched_process_exit")
-int tracepoint__sched_process_exit(void *ctx)
-{
-  u64 pid_tgid = bpf_get_current_pid_tgid();
-  u32 pid      = (u32)(pid_tgid >> 32);
-  u32 tid      = (u32)(pid_tgid & 0xFFFFFFFF);
+struct sched_process_free_ctx {
```

Comment:
Can we add some documentation or information of the origin of this struct?
Either `/sys/kernel/debug/tracing/events/sched/sched_process_free/format` or some link to some other source?